src/app/routes/api/v3/admin/forms/__tests__/admin-forms.form.routes.spec.ts

```diff
@@ -1429,4 +1437,228 @@ describe('admin-form.form.routes', () => {
+    })
+
+    it('should return 401 when user is not logged in', async () => {
+      // Arrange
+      await logoutSession(request)
+
+      const formToUpdate = (await EmailFormModel.create({
+        title: 'Form to update',
+        emails: [defaultUser.email],
+        admin: defaultUser._id,
+        form_fields: [generateDefaultField(BasicField.Date)],
+      })) as IPopulatedForm
+
+      const fieldToDuplicate = formToUpdate.form_fields[0]
+
+      // Act
+      const response = await request.post(
+        `/admin/forms/${formToUpdate._id}/fields/${fieldToDuplicate._id}/duplicate`,
+      )
+
+      // Assert
+      expect(response.status).toEqual(401)
+      expect(response.body).toEqual({ message: 'User is unauthorized.' })
+    })
+
+    it('should return 403 when user does not have permissions to update form', async () => {
+      // Arrange
+      // Create separate user
+      const collabUser = (
+        await dbHandler.insertFormCollectionReqs({
+          userId: new ObjectId(),
+          mailName: 'collab-user',
+          shortName: 'collabUser',
+        })
+      ).user
+
+      const randomForm = (await EncryptFormModel.create({
+        title: 'form that user has no write access to',
+        admin: collabUser._id,
+        publicKey: 'some random key',
+        permissionList: [{ email: defaultUser.email }],
+        form_fields: [generateDefaultField(BasicField.Date)],
+      })) as IPopulatedForm
+
+      const fieldToDuplicate = randomForm.form_fields[0]
+
+      // Act
+      const response = await request.post(
+        `/admin/forms/${randomForm._id}/fields/${fieldToDuplicate._id}/duplicate`,
+      )
+
+      // Assert
+      expect(response.status).toEqual(403)
+      expect(response.body).toEqual({
+        message: `User ${defaultUser.email} not authorized to perform write operation on Form ${randomForm._id} with title: ${randomForm.title}.`,
+      })
+    })
+
+    it('should return 404 when form to update cannot be found', async () => {
+      // Arrange
+      const invalidFormId = new ObjectId()
+      const randomFieldId = new ObjectId()
+
+      // Act
+      const response = await request.post(
+        `/admin/forms/${invalidFormId}/fields/${randomFieldId}/duplicate`,
+      )
+
+      // Assert
+      expect(response.status).toEqual(404)
+      expect(response.body).toEqual({ message: 'Form not found' })
+    })
+
+    it('should return 404 when field to duplicate cannot be found', async () => {
+      // Arrange
+      const formToUpdate = (await EmailFormModel.create({
+        title: 'Form to update',
+        emails: [defaultUser.email],
+        admin: defaultUser._id,
+        form_fields: [generateDefaultField(BasicField.Date)],
+      })) as IPopulatedForm
+
+      const fieldToDuplicate = formToUpdate.form_fields[0]
+      let randomFieldId = new ObjectId()
+
+      while (fieldToDuplicate._id == randomFieldId) {
+        randomFieldId = new ObjectId()
+      }
```

Comment:
will do a more thorough review later, but this isn't necessary because objectIDs are designed to be unique. check out the [docs for ObjectId](https://docs.mongodb.com/manual/reference/method/ObjectId/)